serenata_toolbox/datasets/__init__.py

```diff
@@ -74,7 +74,13 @@ def fetch(filename, destination_path):
     datasets = Datasets(destination_path)
     return datasets.downloader.download(filename)
 
+
 def fetch_latest_backup(destination_path):
     datasets = Datasets(destination_path)
-    files = datasets.downloader.LATEST
+
+    files = tuple(
+        f for f in datasets.downloader.LATEST
+        if not os.path.exists(destination_path, f)
```

Comment:
This needs a little fix, I believe here you need to build the path string since [`os.path.exists()` only takes one argument](https://docs.python.org/3.6/library/os.path.html#os.path.exists) maybe something like: `os.path.exists(os.path.join(destination_path,f))`

Reply:
Thank you @jtemporal. I introduced this error when moving to list comprehension and forgot to test manually after that.